src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -316,7 +302,7 @@ export const ManageCollateralScene = <T extends keyof ParamList>(props: Props<T>
   // #endregion Handlers
 
   return (
-    <FormScene headerText={headerText} onSliderComplete={handleSliderComplete} sliderDisabled={approvalAction == null}>
+    <FormScene headerText={headerText} onSliderComplete={handleSliderComplete} sliderDisabled={actionProgram == null}>
```

Comment:
We should include an insufficient funds case to the `sliderDisabled` condition in a later commit.

Reply:
Agreed, actually forgot to make a TODO on that to expand the UI design for that case.

In any case, it will be handled here: https://app.asana.com/0/1200382638405084/1203187329419620/f